dll/win32/msgina/utils.c

```diff
@@ -94,15 +94,36 @@ ReadRegDwordValue(
 {
     LONG rc;
     DWORD dwValue, dwType, cbData;
-
-    cbData = sizeof(dwValue);
-    rc = RegQueryValueExW(hKey, pszValue, NULL, &dwType, (PBYTE)&dwValue, &cbData);
+    /*
+     * Buffer big enough to hold the NULL-terminated string L"4294967295",
+     * corresponding to the literal 0xFFFFFFFF (MAXULONG) in decimal.
+     */
```

Comment:
Really, do we need 4 lines of comment for this?

Reply:
```suggestion
    /* Buffer big enough to hold the NULL-terminated string L"4294967295",
     * corresponding to the literal 0xFFFFFFFF (MAXULONG) in decimal. */
```